Library/Homebrew/os/mac/linkage_checker.rb

```diff
@@ -49,6 +51,25 @@ def check_dylibs
           end
         end
       end
+
+      next unless OS.linux?
+      system_whitelist = %w[
+        ld-linux-x86-64.so.2
```

Comment:
You'll notice I've added ld-linux-x86-64.so.2 from #342. I expect that one to be merged first since it's less complicated.